entrypoint

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This command takes one positional argument, `command`, which specifies whether
+to initialize test repos locally, or to start an hg serve comand serving the
+test repos.
+"""
+
+import argparse
+import subprocess
+import os
+from pathlib import Path
+
+
+BASE_REPO_DIR = Path("/repos")
+
+HG_SERV_PORT = os.environ.get("PORT", "8000")
+PHABRICATOR_URI = os.environ.get("PHABRICATOR_URI", "http://phabricator.test")
+
+DO_ARCCONFIG = os.environ.get("ARCCONFIG", "").lower() in ("y", "1", "true")
+BASE_ARC_CONFIG = {
+    "phabricator.uri": PHABRICATOR_URI,
+    "history.immutable": False,
+}
+
+# Add or remove entires from this list to create more or less test repos.
+REPOS = [
+    # At the time of writing, test-repo is used in the "dev" instance for testing.
+    "test-repo",
+    # The repos below are used by the local conduit suite (phabricator and lando-api).
+    # If these values are changed here, the local phabricator SQL dump and the hardcoded
+    # repo values in `landoapi.repos` will need to be updated.
+    "first-repo",
+    "second-repo",
+    "third-repo",
+]
+
+# Create a config template for the `hg serve` command.
+WEBDIR_PATHS = "\n".join([f"{r} = {BASE_REPO_DIR/r}" for r in REPOS])
+WEBDIR_CONF = "[paths]\n{}".format(WEBDIR_PATHS)
+WEBDIR_CONF_PATH = BASE_REPO_DIR / "webdir.conf"
+
+
+def init():
+    """Perform initialization on the local dev environment.
+
+    Calls `create_repo` on a pre-defined list of repos, and sets up HG's web config.
+    """
+    for repo in REPOS:
+        print(f"Creating {repo}...")
+        create_repo(repo, DO_ARCCONFIG)
+
+    with WEBDIR_CONF_PATH.open("w") as f:
+        print(f"Copying webdir conf to {WEBDIR_CONF_PATH}...")
+        f.write(WEBDIR_CONF)
+
+
+def start():
+    """Run a local HG web server instance.
+    """
+    subprocess.call(
+        [
+            "hg",
+            "serve",
+            "--port",
+            HG_SERV_PORT,
+            "--accesslog",
+            "/dev/null",
```

Comment:
In v-c-t we usually set both `-A` (aka `--accesslog`) and `-E` (error logs) then print the errors in the tests. This way if something goes wrong with the `hg serve` process we can see a stack trace from inside `hg serve`.

It might not be worth doing here since we're testing code that is essentially a client to the hg server, but I thought I would mention it anyways.

Reply:
Good point. I copied this from the old shell script, but I do think that this can come in handy. Will update.